src/components/PluginModal.astro

```diff
@@ -39,6 +39,9 @@ const authors = new Intl.ListFormat("en", {
       </div>
     )
   }
+  <div class="mb-4">
+    <Code lang="bash" code={`sern plugin ${name}`} />
```

Comment:
is it possible to add a copy paste button as well?

Reply:
will see